system/cfme-setup.sh

```diff
@@ -40,9 +40,6 @@ EOF
 [ -x /sbin/restorecon ] && /sbin/restorecon -R -v /var/www/miq/vmdb/certs
 [ -x /sbin/restorecon ] && /sbin/restorecon -R -v /var/www/miq/system/logrotate_free_space_check.sh
 
-# relabel the pg_log directory in postgresql datadir, but defer restorecon
-# until after the database is initialized during firstboot configuration
-/usr/sbin/semanage fcontext -a -t var_log_t "/opt/rh/postgresql92/root/var/lib/pgsql/data/pg_log(/.*)?"
```

Comment:
Is this something you can test on a nightly appliances before merging?  I'm fine with changing it but we need to ensure the db configuration works and the logrotate of pg logs works.